pkgs/games/atlauncher/default.nix

```diff
@@ -2,11 +2,16 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "atlauncher";
-  version = "3.4.30.0";
+  version = "3.4.34.0";
 
   src = fetchurl {
     url = "https://github.com/ATLauncher/ATLauncher/releases/download/v${finalAttrs.version}/ATLauncher-${finalAttrs.version}.jar";
-    hash = "sha256-eSb+qH/ZoPpeOyIuoJsEpecG7uUyh0vkqZzNuxriRuI=";
+    hash = "sha256-gHUYZaxADchikoCmAfqFjVbMYhhiwg2BZKctmww1Mlw=";
+  };
+
+  icon = fetchurl {
+    url = "https://avatars.githubusercontent.com/u/7068667";
```

Comment:
This URL is not really guaranteed to be stable. The file could change and invalidate the hash. Do they host it on any repo? Since it is in the nixpkgs cache it wont likely break, but still.

Reply:
Hmm, I'll try to find a stable URL for this.